detox/runners/jest/WorkerAssignReporterImpl.js

```diff
@@ -8,8 +9,10 @@ class WorkerAssignReporterImpl extends ReporterBase {
   }
 
   report(workerName) {
-    this._traceln(`${chalk.whiteBright(workerName)} assigned to ${chalk.blueBright(this.device.name)}`);
-    this._traceln('');
+    const deviceName = _.attempt(() => this.device.name);
+    if (!_.isError(deviceName)) {
+      this._traceln(`${chalk.whiteBright(workerName)} assigned to ${chalk.blueBright(this.device.name)}\n`);
+    }
```

Comment:
@noomorph I'd actually want to keep this log even if the device name isn't available because it helps with bug hunting in this area